firmware-binaries/.cargo/config.toml

```diff
@@ -9,4 +9,3 @@ target-dir = "../_build/cargo/firmware-binaries"
 
 [unstable]
 build-std = ["core,panic_abort"]
-build-std-features = ["panic_immediate_abort"]
```

Comment:
It would be great if we could select this based on debug/release, but I don't know if that's possible :pensive:

Reply:
Yeahh, that would be nice. But first we'd need to figure out if we can reduce the Debug size I think